build.gradle.kts

```diff
@@ -58,14 +58,15 @@ tasks {
         withSourcesJar()
     }
     javadoc {
+        options.encoding = "UTF-8"
         options {
             (this as CoreJavadocOptions).addStringOption("Xdoclint:none", "-quiet")
         }
     }
 
     processResources {
         filteringCharset = "UTF-8"
-        filesMatching(listOf("plugin.yml", "config/*.yml")) {
+        filesMatching(listOf("plugin.yml")) {
```

Comment:
This replacement still needs to occur. Didn't know this expansion screwed with escapes so thanks for that